app/policies/wiki_policy.rb

```diff
@@ -1,3 +1,79 @@
 class WikiPolicy < ApplicationPolicy
+  attr_reader :user, :wiki
 
+  def initialize(user, wiki)
```

Comment:
You could have left this method out and just used and record everywhere. Also means you would get rid of the attr_reader on line 2